spec/services/solidus_bolt/users/sync_payment_sources_service_spec.rb

```diff
@@ -8,8 +8,11 @@
   end
 
   let(:user) { create(:user) }
+  let(:bolt_payment_method) { create(:bolt_payment_method) }
 
   describe '#call' do
+    before { bolt_payment_method }
+
```

Comment:
Can you add a check that the payment_method is saved correctly?

Reply:
fixed in https://github.com/nebulab/solidus_bolt/pull/89/commits/aa0211a2a77ec198ac11667e55e3d823900f3083